contracts/contracts/TACoApplication.sol

```diff
@@ -222,6 +242,8 @@ contract TACoApplication is
      * @param _deauthorizationDuration Duration of decreasing authorization in seconds
      * @param _commitmentDurationOptions Options for commitment duration
      * @param _commitmentDeadline Last date to make a commitment
+     * @param _penaltyDefault Default penalty percentage
```

Comment:
Trying to figure out how to make it known/clear that underlying value should be out of 10000 for calcs.
```suggestion
     * @param _penaltyDefault Default penalty percentage (as a value out of 10000)
```

Unless we want to take a value and adjust it to be out of 10000.

Reply:
✔️